### PR TITLE
fixes: integrations

### PR DESCRIPTION
**Pull request by tynes**
## Description

Update deps explicitly, return `l1BlockNumber` as a JS number instead of hex string

### Fixes
- Fixes # [Link to Issue]

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "release:beta": "yarn clean && yarn build && lerna version prerelease --yes && lerna publish from-package --yes --force-publish --dist-tag=beta -m \"chore(@ethereum-optimism) publish %s release\"",
     "patch": "yarn run patch:ganache && yarn run patch:waffle",
     "patch:ganache": "rm node_modules/ganache-core/typings/index.d.ts ||:",
-    "patch:waffle": "sed -ie 's!Ganache.GanacheOpts!any!g' node_modules/ethereum-waffle/dist/waffle.d.ts"
+    "patch:waffle": "sed -ie 's!Ganache.GanacheOpts!any!g' node_modules/ethereum-waffle/dist/waffle.d.ts ||:"
```
Comment (kfichter) [inline package.json:34]: :+1:

**Review by karlfloersch — APPROVED** [review]
LGTM!

File: packages/provider/src/app/provider.ts [hunk 2]
```diff
@@ -69,6 +69,9 @@ export class OptimismProvider extends JsonRpcProvider {
       b.stateRoot = block.stateRoot
       for (let i = 0; i < b.transactions.length; i++) {
         b.transactions[i].l1BlockNumber = block.transactions[i].l1BlockNumber
+        if (b.transactions[i].l1BlockNumber != null) {
```
Comment (kfichter) [inline packages/provider/src/app/provider.ts:72]: What happens if this is null?

File: packages/provider/src/app/provider.ts [hunk 3]
```diff
@@ -69,6 +69,9 @@ export class OptimismProvider extends JsonRpcProvider {
...
+          b.transactions[i].l1BlockNumber = parseInt(b.transactions[i].l1BlockNumber, 16)
```
Comment (karlfloersch) [inline packages/provider/src/app/provider.ts:73]: This should be set to the l1BlockNumber even for sequencer transactions 😭

**Pull request merged**